pkgs/top-level/python-packages.nix

```diff
@@ -10552,12 +10552,29 @@ in {
   };
 
   msgpack = buildPythonPackage rec {
+    name = "msgpack-${version}";
+    version = "0.5.4";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/m/msgpack/${name}.tar.gz";
+      sha256 = "13ckbs2qc4dww7fddnm9cw116j4spgxqab49ijmj6jr178ypwl80";
+    };
+
+    checkPhase = ''
+      py.test
+    '';
+
+    buildInputs = with self; [ pytest ];
+    propagatedBuildInputs = with self; [ ];
+  };
+
+  msgpack-transitional = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.